### PR TITLE
Configure flags

### PR DESCRIPTION
**Pull request by AlexanderPrendota**

### CODE REVIEW

File: plugin/src/main/kotlin/com/kotlin/aws/runtime/tasks/ConfigureGraal.kt [hunk 1]
```diff
@@ -165,13 +160,32 @@ internal object ConfigureGraal {
         return file
     }
 
-    //TODO probable reflect.json should be configurable and we should have few preconfigured
-    private fun generateReflect(buildDir: File): File {
-        val reflect = ConfigureGraal::class.java.getResource("/${GraalSettings.DEFAULT_REFLECT_FILE_NAME}").readText()
+    private fun Project.getGraalVmImage(): String {
+        val image = runtime.config.image ?: GraalSettings.GRAAL_VM_DOCKER_IMAGE
```
Comment (TanVD) [inline plugin/src/main/kotlin/com/kotlin/aws/runtime/tasks/ConfigureGraal.kt]: I would better move it to `Runtime` as internal function. like `internal fun RuntimeConfig.getImageOrDefault()`

File: plugin/src/main/kotlin/com/kotlin/aws/runtime/tasks/ConfigureGraal.kt [hunk 2]
```diff
@@ -165,13 +160,32 @@ internal object ConfigureGraal {
...
+        logger.lifecycle("Create GraalVM native image: `$image`.")
+        return image
+    }
+
+    private fun Project.getGraalVmFlags(): String {
```
Comment (TanVD) [inline plugin/src/main/kotlin/com/kotlin/aws/runtime/tasks/ConfigureGraal.kt]: Same here :)

File: plugin/src/main/kotlin/com/kotlin/aws/runtime/tasks/GenerateAdapter.kt [hunk 3]
```diff
@@ -23,9 +23,11 @@ open class GenerateAdapter : DefaultTask() {
 
     @TaskAction
     fun act() {
-        val (klass, function) = handler?.split("::")
-            ?: error("`handler` field should be set via `runtime` extension")
-
+        val handler = handler?.split("::")
+        if (handler == null || handler.size != 2) {
```
Comment (TanVD) [inline plugin/src/main/kotlin/com/kotlin/aws/runtime/tasks/GenerateAdapter.kt]: Transform to `require` -- `require(handler != null && handler.size == 2`) { .... }`

**Review by TanVD — CHANGES_REQUESTED** [review]

File: plugin/src/main/kotlin/com/kotlin/aws/runtime/tasks/GenerateAdapter.kt (hunk 3 above)
Comment (AlexanderPrendota) [reply plugin/src/main/kotlin/com/kotlin/aws/runtime/tasks/GenerateAdapter.kt]: yep make sense

**Review by TanVD — APPROVED** [review]

**Pull request merged**